tutorials/developer-tutorials/createNewIdentityAttributes.md

```diff
@@ -0,0 +1,168 @@
+
+As an example, suppose we have a smart contract which allows participants to record the cars owned by a company on a blockchain. We may allow participants to add cars to the blockchain by submitting a `createCar` transaction. Now, this would be fine for `Arium` because it is a manufacturer, but since `Jack` is an individual, if he tries to submit a `createCar` transaction, we should stop him! Here’s how you would register and enrol an identity for `Arium` from the Local Fabric CA with suitable attributes…
+
+1. Under the `FABRIC ENVIRONMENTS` panel, click on `Local Fabric  ○ (click to start)` to start the local fabric and connect to it. Once this is done, look for `ca.org1.example.com` (it's under Nodes), right click it and choose `Create Identity (register and enroll)`. 
+
+2. You will be asked to provide a name for your identity. For the purpose of this tutorial, we will call our identity `Arium` and select `Yes` when asked to add attributes. We will give Arium the attributes `[{"name": "manufacturer", "value": "true", "ecert":true}]`
+
+>`ecert` stands for Enrollement Certificate. When set to true, it writes the attributes into the identity's certificate. This allows our createCar transaction to read the attributes of the identity that invokes this transaction and decide whether or not it is allowed to carry out the transaction (based on it's attributes). If the ecert isn't set to true, we cannot implement attribute-based access control.
+
+3. Upon submitting your request, you should see a dialog box at the bottom right of the screen confirming that your identity `Arium` has been created with attributes `[{"name": "manufacturer", "value": "true", "ecert":true}]`. The newly created identity should also appear in the `Fabric Wallets` panel under `Local Fabric Wallet`. 
+
+4. To connect to the Local Fabric gateway using this identity, you simply click `Local Fabric` under the Fabric Gateways panel and select the identity you wish to connect with (Arium in our case).
+> Command Palette alternative: `Create Identity (register and enroll)`
+</details>
+
+<details>
+<summary><b>3. Using our new identity</b></summary>
+
+Now that we've created our very own identities, we can implement attribute-based access control. To do so, follow the steps below:
+
+1. Click the `Explorer` icon in the left sidebar (it will be at the top and will look like a folder) and navigate to the `car-contract.ts` file in the `src` folder under the contract directory.
+
+2. To deny Jack from carrying out the `createCar` transaction, we need to tweak the generated smart contract slightly. We can replace the function with the following: 
+<details open="true">
```

Comment:
You don't need this as you only have the code in typescript and not in java as well